README.md

```diff
@@ -27,42 +27,79 @@ const Example = () => (
 
 This repo has to be run with https://github.com/openworklabs/aragon
 
-Clone this repo, install dependencies, link, and build: <br />
-`git clone git@github.com:AutarkLabs/aragon-profile.git`<br />
-`cd aragon-profile`<br />
-`npm i`<br />
-`npm run prepare-local-dev`<br />
-`npm link`<br />
-`npm build`<br />
+Clone this repo, install dependencies, link, and build:
+`git clone git@github.com:AutarkLabs/aragon-profile.git`
+`cd aragon-profile`
+`npm i`
+`npm run prepare-local-dev`
+`npm link`
+`npm run build`
 
 After finishing these commands, your `aragon-profile` should have a `/dist` directory with the ES5 code.
 
 **NOTE** - anytime you want to see your changes on the aragon-client, you have to re-run `npm run build`
 
-In a separate repository location, clone the aragon client repo: <br />
+In a separate repository location, clone the aragon client repo:
 `git clone git@github.com:openworklabs/aragon.git`
 
-Get the development branch of the aragon client repo:<br />
-`cd aragon`<br />
+Get the development branch of the aragon client repo:
+`cd aragon`
```

Comment:
Same thing about line breaks here and throughout.

Also here: Should we add a line for which branch to checkout? "Get the development branch", but I don't think that branch is the default. Is `dev` the correct branch?